.github/workflows/gh-actions-pr.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       fail-fast: true
       matrix:
         include:
+          - FROM:     'ubuntu:impish'
+            COMPILER: 'gcc'
+            PREFER_LEGACY_OPENGL: 'OFF'
```

Comment:
@romanyacik we also need a Clang version as we use multiple compilers. Use the `hirsute` as an example.
@stephengtuggy do we need both version of PREFER_LEGACY_OPENGL? Or only one version?

Reply:
Yes, I would say we need all four combinations of gcc vs. clang, and PREFER_LEGACY_OPENGL `ON` vs. `OFF`.

Before any of that will work, though, we need to add Impish to the bootstrap scripts; to CMakeLists.txt; and to the CI/CD for `build-system-docker-images`.